### PR TITLE
Conductor update

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Changes:
- conductor updated from 2.2.0 to 2.11.0 (based conductor image 2.11.0 has been built from Netflix/conductor 2.11.0 tag and pushed to dockerhub manually) 
- tasks definitions fixed (conductor 2.11.0 will not work with existing definitions)
- additional timeouts in conductor simple HTTP tasks introduced
- default configuration attached to the image
- startup script modified to accept logging config file and java options

Issues: https://tracker.mender.io/browse/MEN-2644

### CODE REVIEW

**Review by mzedel — APPROVED** [review]
I trust that the changes work as expected 🙂
From the code alone I'm wondering if the `config.properties` requires the dynomite options to be present, since only redis should be used - if I'm not mistaken.
If possible I would get rid of all the rest, but considering that SRE might want to run it in HA anyway - it's probably not a bad idea to have it in place as documentation.

File: server/config/config.properties [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#
+# memory : The data is stored in memory and lost when the server dies.  Useful for testing or demo
+# redis : non-Dynomite based redis instance
+# dynomite : Dynomite cluster.  Use this for HA configuration.
+
+db=redis
+
+# Dynomite Cluster details.
+# format is host:port:rack separated by semicolon
+workflow.dynomite.cluster.hosts=mender-redis:6379:us-east-1b
```
Comment (0lmi) [inline server/config/config.properties:12]: It's important to have ability to change parameters like this. I believe we already have such pattern used in api-gateway service, when we have some special 'vars' like [@LOGS_FORMAT@](https://github.com/mendersoftware/mender-api-gateway-docker/blob/master/nginx.conf#L35) for instance, which are replaced by entrypoint script. This gives ability to configure service for different environments (dev/staging/prod) by setting environment variables.

**Review by 0lmi — CHANGES_REQUESTED** [review]

File: server/config/config.properties (hunk 1 above)
Comment (kjaskiewiczz) [reply server/config/config.properties:12]: I do not fully understand what kind of change you are requesting to this PR.
Should this be a part of conductor update?
Or can you create a separate task for it?

Comment (0lmi) [reply server/config/config.properties:12]: Yeah, agree, it makes sense to move it to separate task.

**Review by 0lmi — APPROVED** [review]

**Pull request merged**